Makefile.dapper

```diff
@@ -1,19 +1,8 @@
-# This Makefile contains the rules required to set up our
-# Dapper-based build environment; it can be copied as-is to
-# other projects (and needs to be copied, it can't be shared
-# via the Dapper image since it's needed to retrieve the image)
+# This Makefile imports the rules to run in a Dapper-based build environment.
+# The rules are being imported via download from Shipyard where the base file can be found
 
-.dapper:
-	@echo Downloading dapper
-	@curl -sL https://releases.rancher.com/dapper/latest/dapper-`uname -s`-`uname -m` > .dapper.tmp
-	@@chmod +x .dapper.tmp
-	@./.dapper.tmp -v
-	@mv .dapper.tmp .dapper
+BASE_BRANCH := devel
+_ := $(shell test -f Makefile.shipyard || curl -L https://raw.githubusercontent.com/submariner-io/shipyard/$(BASE_BRANCH)/Makefile.dapper > Makefile.shipyard)
```

Comment:
How do we ensure that local builds don’t use a stale file?

Reply:
Well the intention was to minimize local redownload each time, however I see were also fetching dapper each time.. So.. Maybe it's unnecessary to optimize right now?

Reply:
But generally what I thought of is to have `make clean` which also removes these cached files, theoretically :)

Reply:
Right, `make clean` would be nice to have! (In another PR, not necessarily by you.)